src/Pim/Bundle/CatalogBundle/Resources/config/completeness_checkers.yml

```diff
@@ -3,28 +3,33 @@ parameters:
     pim_catalog.completeness.checker.media.class:   Pim\Component\Catalog\Completeness\Checker\MediaCompleteChecker
     pim_catalog.completeness.checker.metric.class:  Pim\Component\Catalog\Completeness\Checker\MetricCompleteChecker
     pim_catalog.completeness.checker.price.class:   Pim\Component\Catalog\Completeness\Checker\PriceCompleteChecker
-    pim_catalog.completeness.checker.empty.class:   Pim\Component\Catalog\Completeness\Checker\EmptyChecker
+    pim_catalog.completeness.checker.simple.class:  Pim\Component\Catalog\Completeness\Checker\SimpleCompleteChecker
 
 services:
     pim_catalog.completeness.checker.chained:
         class: '%pim_catalog.completeness.checker.chained.class%'
-
-    pim_catalog.completeness.checker.empty:
-        class: '%pim_catalog.completeness.checker.empty.class%'
-        tags:
-            - { name: pim_catalog.completeness.checker.product_value, priority: 80 }
+        public: false
 
     pim_catalog.completeness.checker.media:
         class: '%pim_catalog.completeness.checker.media.class%'
+        public: false
         tags:
             - { name: pim_catalog.completeness.checker.product_value, priority: 100 }
 
     pim_catalog.completeness.checker.metric:
         class: '%pim_catalog.completeness.checker.metric.class%'
+        public: false
         tags:
             - { name: pim_catalog.completeness.checker.product_value, priority: 100 }
 
     pim_catalog.completeness.checker.price:
         class: '%pim_catalog.completeness.checker.price.class%'
+        public: false
         tags:
             - { name: pim_catalog.completeness.checker.product_value, priority: 100 }
+
+    pim_catalog.completeness.checker.simple:
+        class: '%pim_catalog.completeness.checker.simple.class%'
```

Comment:
I don't like "simple", is it a PIM convention naming? does it only check completeness for scalar value?